content/01.abstract.md

```diff
@@ -11,3 +11,21 @@ Publication with Manubot is continuous:
 When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
 Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers (e.g. DOIs, PubMed IDs, ISBNs, URLs), whose metadata is then retrieved and converted to a user-specified style.
 Manubot modernizes publishing to align with the ideals of open science by making it transparent, reproducible, immediate, versioned, collaborative, and free of charge.
+
+## Author summary
+
+Traditionally, scholarly manuscripts, such as this one you are reading, have been written in private.
+This worked because the research was also performed in private by a predefined team of collaborators.
```

Comment:
Do we need to introduce the idea that typically a fairly small number of authors wrote the manuscript in this setting here?  It may be sufficiently covered in the next paragraph.